clients/src/main/java/org/apache/kafka/common/errors/GroupIdNotFoundException.java

```diff
@@ -17,15 +17,7 @@
 package org.apache.kafka.common.errors;
 
 public class GroupIdNotFoundException extends ApiException {
-    private final String groupId;
-
-    public GroupIdNotFoundException(String groupId) {
-        super("The group id " + groupId + " was not found");
-        this.groupId = groupId;
-    }
-
-    public String groupId() {
-        return groupId;
+    public GroupIdNotFoundException(String message) {
```

Comment:
Hmm.. This is technically a public api as of 1.1, though I don't think it was raised from any API in that release. Perhaps it is justifiable since the expectation was never correctly implemented anyway.

Reply:
That's a fair point.  The API could never work, though (and still never can).  ApiExceptions need to be deserializable from Error enums without additional information.  Additional information is a nice-to-have, but may or may not be provided.  If anyone tried to use this in 1.1, it wouldn't work.

Reply:
Yeah, I guess I'm ok with it since the API couldn't work as intended.